bin/helpers/load_environment.sh

```diff
@@ -4,4 +4,7 @@ if [ ! -f .env ]; then
     printf "\e[0;31m%s\e[0m\n" "Environment file must be set!"
     exit 1
 fi
-source .env
\ No newline at end of file
+source .env
+
+TRAVIS_COMMIT="$(git rev-parse --short HEAD 2> /dev/null | sed "s/\(.*\)/\1/")"
+TRAVIS_BRANCH="$(git symbolic-ref --short -q HEAD)"
```

Comment:
Travis CI has it's own target branch exposing logic. Why are you are overriding this?
At least put both variables in conditional logc if its not set - set to default value